src/app/(afterlogin)/dashboard/page.tsx

```diff
@@ -1,3 +1,168 @@
+'use client';
+
+import Navigation from '@/app/_components/nav/Navigation';
+import ProgressBar from '@/app/_components/tasks/ProgressBar';
+import TaskListItem from '@/app/_components/tasks/TaskListItem';
+import Image from 'next/image';
+import { useEffect, useState } from 'react';
+
 export default function Dashboard() {
-  return <div>대시보드</div>;
+  const [currentTime, setCurrentTime] = useState('');
+  const [today, setToday] = useState('');
+  const [mockData, setMockData] = useState([
+    {
+      task_id: 1,
+      title: '일어나기',
+      memo: '',
+      start_time: '2025-03-11T13:00:00',
+      end_time: '',
+      address: '경기도 수원시 ...',
+      place_name: 'Cafe ABC',
+      location: { lat: '', lng: '' },
+      is_completed: true,
+    },
+    {
+      task_id: 2,
+      title: 'Team meeting',
+      memo: '주간 회의',
+      start_time: '2025-03-11T13:00:00',
+      end_time: '',
+      address: '경기도 수원시 ...',
+      place_name: 'Zep 4번 룸',
+      location: { lat: '127.1086228', lng: '37.4012191' },
+      is_completed: false,
+    },
+    {
+      task_id: 3,
+      title: '구현하기',
+      memo: '수요일까지 구현해요',
+      start_time: '2025-03-11T13:00:00',
+      end_time: '2025-03-11T14:00:00',
+      address: '경기도 수원시 ...',
+      place_name: 'Cafe ABC',
+      location: { lat: '127.1086228', lng: '37.4012191' },
+      is_completed: false,
+    },
+  ]);
+  const [finishedTaskCount, setFinishedTaskCount] = useState(0);
+
+  const getCurrentTime = () => {
+    const currentDate = new Date();
+    const timeString = Intl.DateTimeFormat('en-US', {
+      hour: '2-digit',
+      minute: '2-digit',
+      hour12: true,
+    }).format(currentDate);
+
+    setCurrentTime(timeString);
+  };
+
+  const handleCheckClick = (taskId: number) => {
+    setMockData(prev =>
+      prev.map(task =>
+        task.task_id === taskId
+          ? { ...task, is_completed: !task.is_completed }
+          : task,
+      ),
+    );
+  };
+
+  useEffect(() => {
+    getCurrentTime();
+    const interval = setInterval(() => {
+      getCurrentTime();
+    }, 1000);
+
+    return () => clearInterval(interval);
+  });
```

Comment:
```suggestion
  useEffect(() => {
    getCurrentTime();
    const interval = setInterval(() => {
      getCurrentTime();
    }, 1000);

    return () => clearInterval(interval);
  }, []);
```
마지막에 의존성 배열을 추가해주면 좋을 것 같습니다 
의존성 배열이 빈 배열이라도 선언해주지 않으면 컴포넌트가 리렌더링 될 때마다 useEffect가 출력됩니다!